pkg/dashboard/test/server_test.go

```diff
@@ -1040,6 +1041,17 @@ func (suite *projectTestSuite) TestExportProjectSuccessful() {
 	returnedProject1.ProjectConfig.Meta.Name = "p1"
 	returnedProject1.ProjectConfig.Meta.Namespace = "fNamespace"
 
+	returnedAPIGateway1 := platform.AbstractAPIGateway{}
+	returnedAPIGateway1.APIGatewayConfig.Meta.Name = "agw1"
+	returnedAPIGateway1.APIGatewayConfig.Meta.Namespace = "fNamespace"
```

Comment:
capitals letters in namespace? this is all mocked so it's cool? if it's k8s it will fail - must be kebab-case